templates/_header.html

```diff
@@ -15,9 +15,7 @@
       </span>
       <ul class="p-navigation__links" role="menu">
         <li class="p-navigation__link" role="menuitem">
-          <a
-            {% if (path and (path.startswith('/store') or path.startswith('/search'))) or snap_title %} class="is-selected"{% endif %}
-            href="/store">Store
+          <a{% if page_slug == 'store' %} class="is-selected"{% endif %} href="/store">Store
```

Comment:
It would read a bit better if the conditional statement was multiline (as with the dropdown below).

Reply:
Nice - done.